Automation_Tools/Web Scrapping/Train Schedule Scraper/fetch_train_schedules.py

```
@@ -0,0 +1,162 @@
+    options.add_argument("--disable-dev-shm-usage")
+    driver = webdriver.Chrome(service=Service(ChromeDriverManager().install()), options=options)
+    return driver
+
+# Function to close login/signup pop-up
+def close_popup(driver):
+    try:
+        close = WebDriverWait(driver, 10).until(
+            EC.element_to_be_clickable((By.XPATH, "/html/body/div/div/div[2]/div[1]"))
+        )
+        close.click()
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)